### PR TITLE
[Lang.ai] Update Connector Upload New files

### PR DESCRIPTION
**Pull request by evicentem9**
---
When submitting a connector, please check the following conditions for your PR to ensure a smooth certification process.

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] Independent Publisher Connectors Only: Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] ndependent Publisher Connectors Only: Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

File: certified-connectors/Lang-ai/apiDefinition.swagger.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
                         "items": {
                           "type": "string"
                         },
-                        "description": "An array of features. The sign “>” indicates a second-level feature."
+                        "description": "An array of features. The sign \u201c>\u201d indicates a second-level feature."
```
Comment (sriyen-msft) [inline certified-connectors/Lang-ai/apiDefinition.swagger.json:66]: @evicentem9 Unicodes are not allowed as they would not be supported in all the Power Automate pages, Can you please let us know the reason to use unicodes here?

File: certified-connectors/Lang-ai/apiDefinition.swagger.json [hunk 2]
```diff
@@ -163,7 +163,7 @@
                         "items": {
                           "type": "string"
                         },
-                        "description": "The list of features. The sign “>” indicates a second-level feature."
+                        "description": "The list of features. The sign \u201c>\u201d indicates a second-level feature."
```
Comment (sriyen-msft) [inline certified-connectors/Lang-ai/apiDefinition.swagger.json:166]: @evicentem9 Unicodes are not allowed as they would not be supported in all the Power Automate pages, Can you please let us know the reason to use unicodes here?

File: certified-connectors/Lang-ai/apiProperties.json [hunk 3]
```diff
@@ -13,12 +13,30 @@
...
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Tenant",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('tenant').lang.ai"
+        }
+      }
+    ],
+    "publisher": "Eduardo Vicente"
```
Comment (sriyen-msft) [inline certified-connectors/Lang-ai/apiProperties.json:40]: @evicentem9 Can you please add `stackOwner` manually

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Lang-ai/apiDefinition.swagger.json (hunk 1 above)
Comment (evicentem9) [reply certified-connectors/Lang-ai/apiDefinition.swagger.json:66]: this was an error, this should be changed. The original description without unicodes is correct.

File: certified-connectors/Lang-ai/apiDefinition.swagger.json (hunk 2 above)
Comment (evicentem9) [reply certified-connectors/Lang-ai/apiDefinition.swagger.json:166]: this was an error, this should be changed. The original description without unicodes is correct.

File: certified-connectors/Lang-ai/apiProperties.json (hunk 3 above)
Comment (evicentem9) [reply certified-connectors/Lang-ai/apiProperties.json:40]: yes, I'll do.

**Review by evicentem9 — COMMENTED** [review]
Already changed

**Review by evicentem9 — COMMENTED** [review]
Already changed

**Comment by evicentem9** [comment]
@sriyen-msft Hi, just want to make sure that I have done the changes in a correct way and you can see them. It's still the status of "Changes requested" but I've already made them.

File: certified-connectors/Lang-ai/apiProperties.json [hunk 4]
```diff
@@ -13,12 +13,31 @@
...
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Tenant",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('tenant').lang.ai"
+        }
+      }
+    ],
+    "publisher": "Eduardo Vicente",
```
Comment (sriyen-msft) [inline certified-connectors/Lang-ai/apiProperties.json:40]: @evicentem9 Sorry for late review, but for publisher and stackOwner properties - individuals are not allowed. Can you please provide only company or organization name. Rest everything looks good.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Lang-ai/apiProperties.json (hunk 4 above)
Comment (evicentem9) [reply certified-connectors/Lang-ai/apiProperties.json]: @sriyen-msft Thanks for the response. Already changed.

**Comment by evicentem9** [comment]
@sriyen-msft Hi!! Sorry for the insistence, but I have already made changes and I want to be sure everything is correct, because I'm pretty new using Github. Thanks in advance.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**